### PR TITLE
Styled the Register Club Button in NavBar

### PR DESCRIPTION
**Pull request by rgu0114**
Screen Shot 2022-04-30 at 11 07 09 AM
^screenshot of new changes 4/29

New changes 4/29: Styled register club button in nav bar so that the body of the button was white, and the text/border was colored. Also updated the sizing and positioning so that it's perfectly in line with the account button and has better distance relative to the top of the screen. 

What remains to be done: need to find some way to ensure that the buttons don't collide with the search bar when the window is shrunk horizontally. Add some sort of wall between the right side of the search bar and the left side of the buttons so that they don't stick together even when the window is tiny. 

Screen Shot 2022-03-19 at 4 52 39 PM
^screenshot of what the changes look like above 
- Created CSS styling for Register Button, setting the background color to salmon (closest color I could find to the designs) and text color to white. Also adjusted the sizing/border radius to follow designs as closely as possible.
- Changed the “right” CSS NavBar class to have display: flex so that the Register Club and Profile buttons didn’t go on two separate lines when the window was narrowed. Decreased the margin-right of this class to make the two buttons appear a bit more to the right.
- wrapped the entire button in a link to provide on click functionality

### Summary 

This pull request is the first step towards styling the Register Club button

- [x] styled button (color, corners, etc)
- [x] linked 

### Test Plan 

I did the npm start simulator thing and the styling appears to be working. Also retains its shape/functionality with different window sizes.

### Notes 

I'm not sure if this conflicts or not with the changes that have taken place since I cloned the repo. Thorough review requested 🙏

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: frontend/src/components/NavBar/NavBar.css [hunk 1]
```diff
@@ -33,15 +33,26 @@
 }
 
 .right {
+  display: flex;
```
Comment (20austinw) [inline frontend/src/components/NavBar/NavBar.css:67]: You should convert px to rem so the sizes of your components scale with the user's font size.

**Review by 20austinw — DISMISSED** [review]
Overall, the CSS looks pretty good. I think it would be beneficial to the reviewer if you also included a screenshot of the end result.

File: frontend/src/components/NavBar/NavBar.css (hunk 1 above)
Comment (20austinw) [reply frontend/src/components/NavBar/NavBar.css:67]: Remember to do "npm run format" before you push the code.

Comment (rgu0114) [reply frontend/src/components/NavBar/NavBar.css:67]: Thanks for the feedback! What does rem do? I tried changing it from px to rem but I think I broke something lol

Comment (20austinw) [reply frontend/src/components/NavBar/NavBar.css:67]: I believe 'rem' is a unit relative to the root element's font size, so if the user scales the font, everything else would get scaled properly. This was something my TPM told me last semester. You can read more about it [here](https://engageinteractive.co.uk/blog/em-vs-rem-vs-px). Remember to [convert](https://nekocalc.com/px-to-rem-converter) your px to rem, you cannot simply change the unit.

Comment (rgu0114) [reply frontend/src/components/NavBar/NavBar.css:67]: Finished converting px -> rem

**Review by JustKong13 — APPROVED** [review]
All the changes look good, follows the figma very well, clean design and clean code! Good work!

**Review by JustKong13 — DISMISSED** [review]
Code looks good, but there seems to be some checks that are failing. Are you able to figure those out?

**Review by justinkang01 — APPROVED** [review]
nice job richard! looking really good  =D

**Comment by rgu0114** [comment]
> nice job richard! looking really good =D

thx dude

**Pull request merged**